todoman/cli.py

```diff
@@ -332,6 +351,13 @@ def list(
 
     sort = sort.split(',') if sort else None
 
+    if start: 
```

Comment:
I'd prefer if we pass the list directly into `db.todos()`. So no extra `before` parameter.